common/test/src/IO/WorldReaderTest.cpp

```diff
@@ -47,6 +48,26 @@ namespace TrenchBroom {
             return nullptr;
         }
 
+        inline void
```

Comment:
Our function declarations usually all go into one line, so return type and `inline` should not get an extra line. Or am I now contradicting what you were saying @ericwa ?

Reply:
Was working from the example of the existing findFaceByPoints function above. I can change all of them to be of the form `inline footype barfunc( ... ) {`

Reply:
Sorry, Yeah. I sometimes write two line like that so it slipped by me, but I guess our standard is one line.

Reply:
Ok, let's change them to one line then. @ericwa if we ever decide to use clang-format to autoformat the codebase, we can have fistfights over the rules of when to put stuff on new lines ;-)